samples/custom_dispatch/cpu/plugin/standalone_ukernel.c

```diff
@@ -0,0 +1,156 @@
+// Copyright 2023 The IREE Authors
+//
+// Licensed under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+// Demonstrates a standalone plugin calling a `simple_mul_workgroup`
+// micro-kernel function. `standalone_ukernel.mlir` shows the input
```

Comment:
Could add a section to https://github.com/openxla/iree/blob/main/samples/custom_dispatch/cpu/plugin/README.md explaining the different sample versions now in this directory (features, benefits/tradeoffs)

Reply:
I dont think I need to update the README now. The readme is just describing how the plugin is to be used. THis is just two different tests.